CHANGELOG.md

```diff
@@ -10,11 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Unreleased]
 ------------
 
+[v0.8.0] - 2020-01-22
+---------------------
+
 ### Added
 
 - Support for backtracing in debug and release builds.
     - Implementations for GNU functions `backtrace` and `backtrace_symbols` (defined in execinfo.h)
     - Enclaves are built using `-fno-omit-frame-pointer` for accurate backtraces.
+- Support for custom attestation data formats via new plugin model. Please refer to [design documentation](docs/DesignDocs/CustomAttestation.md).
+- Support for host side sockets on Windows.
```

Comment:
There seems to be no mention of sockets for linux elsewhere. Are sockets an internal feature?

Reply:
Sockets are not internal as enclave apps can call call the POSIX socket APIs. You are right on Linux support not being mentioned. It perhaps was an oversight.